pkg/features/kube_features.go

```diff
@@ -60,7 +61,7 @@ func init() {
 // Entries are separated from each other with blank lines to avoid sweeping gofmt changes
 // when adding or removing one entry.
 var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	PartialAdmission:  {Default: false, PreRelease: featuregate.Alpha},
+	PartialAdmission:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
This is pretty much the only change necessary in the production code.

Maybe we could add a simple e2e test using Job.

Reply:
I know.

e2e added